test_requirements.txt

```diff
@@ -0,0 +1 @@
+pytest>=4.3.1,<6.0
```

Comment:
I tested the tests pass with v4.31 and the latest v.5.2.4

Reply:
@ChristopherGS I don't have enough words to thank you!!!!
You are amazing. Thank you for so much work.